Gemfile

```diff
@@ -3,6 +3,7 @@ source 'https://rubygems.org'
 gem 'slop'
 gem 'rake'
 gem 'zeitwerk'
+gem 'ruby_dig'
```

Comment:
`dig` is part of the `Hash` stdlib

Reply:
If I removed this line I got the following error:

yinfangzhuang@Yinfangs-MacBook-Pro test % bundle exec rake test
(in /Users/yinfangzhuang/aurora-bootstrap-parallelization)
Zeitwerk@272fa9: autoload set for AuroraBootstrapParallelization, to be autovivified from /Users/yinfangzhuang/aurora-bootstrap-parallelization/lib/aurora_bootstrap_parallelization
Zeitwerk@272fa9: module AuroraBootstrapParallelization autovivified from directory /Users/yinfangzhuang/aurora-bootstrap-parallelization/lib/aurora_bootstrap_parallelization
Zeitwerk@272fa9: autoload set for AuroraBootstrapParallelization::JobExporter, to be loaded from /Users/yinfangzhuang/aurora-bootstrap-parallelization/lib/aurora_bootstrap_parallelization/job_exporter.rb
Zeitwerk@272fa9: autoload set for AuroraBootstrapParallelization::Exporter, to be loaded from /Users/yinfangzhuang/aurora-bootstrap-parallelization/lib/aurora_bootstrap_parallelization/exporter.rb
Zeitwerk@272fa9: autoload set for AuroraBootstrapParallelization::CronjobExporter, to be loaded from /Users/yinfangzhuang/aurora-bootstrap-parallelization/lib/aurora_bootstrap_parallelization/cronjob_exporter.rb
Zeitwerk@272fa9: autoload set for AuroraBootstrapParallelization::Runner, to be loaded from /Users/yinfangzhuang/aurora-bootstrap-parallelization/lib/aurora_bootstrap_parallelization/runner.rb

**_File does not exist: ruby_dig_**

rake aborted!
Command failed with status (1)
/Users/yinfangzhuang/.asdf/installs/ruby/2.6.6/bin/bundle:23:in `load'
/Users/yinfangzhuang/.asdf/installs/ruby/2.6.6/bin/bundle:23:in `<main>'
Tasks: TOP => test
(See full trace by running task with --trace)

Reply:
You are right here.